src/platform/GenericOTARequestorDriver.cpp

```diff
@@ -58,7 +58,16 @@ void GenericOTARequestorDriver::UpdateAvailable(const UpdateDescription & update
 
 void GenericOTARequestorDriver::UpdateNotFound(UpdateNotFoundReason reason, System::Clock::Seconds32 delay)
 {
-    // TODO: Schedule the next QueryImage
+    VerifyOrDie(mRequestor != nullptr);
+
+    System::Clock::Seconds32 delayedActionTime = System::Clock::Seconds32(120);
```

Comment:
Is this 120 from spec? If yes, can we use some macro here so that it is understood?

Reply:
Done, PTAL!